app/main.py

```diff
@@ -1,8 +1,51 @@
 class Car:
-    # write your code here
-    pass
+
+    def __init__(self: None, comfort_class: None,
+                 clean_mark: None,
+                 brand: None) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self: None, distance_from_city_center: None,
+                 clean_power: None,
+                 average_rating: None,
+                 count_of_ratings: None) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self: None, cars: None) -> None:
```

Comment:
Do not use type annotation for `self` here and in other parts of your code. Indicate that you expect to obtain a list of `Car` instances here for `cars`.